CONTRIBUTING_PACKAGES.md

```diff
@@ -40,7 +40,7 @@ Before you develop your custom Great Expectations package, notify the Great Expe
 
 ## Install the Great Expectations CLI tool
 
-Great Expectations provides the `great_expectations_contrib` command-line interface (CLI) tool to simplify the package contribution process and help you adhere to code base best practices. 
+Great Expectations provides the `great_expectations_contrib` command-line interface (CLI) tool to simplify the package contribution process and help you adhere to code base best practices.
```

Comment:
It is not clear why this text appears as changed -- it seems identical to what was there before, save for the blank space at the end (result of linter).

Reply:
Yup it's a linter rule to clean up excess whitespace